pages/ExitIframe.jsx

```diff
@@ -0,0 +1,20 @@
+import { Redirect } from "@shopify/app-bridge/actions";
+import { useAppBridge, Loading } from "@shopify/app-bridge-react";
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+export default function ExitIframe() {
+  const app = useAppBridge();
+  const { search } = useLocation();
+
+  useEffect(() => {
+    const params = new URLSearchParams(search);
+    const redirect = Redirect.create(app);
+    redirect.dispatch(
+      Redirect.Action.REMOTE,
+      decodeURIComponent(params.get("redirectUri"))
+    );
+  });
```

Comment:
```suggestion
  }, [app, search]);
```

`useEffect` should always include a dependency array, but you might need a conditional in the callback to check for the presence of `app` and `search`?

Reply:
In any case, we should add https://www.npmjs.com/package/eslint-plugin-react-hooks to this library.

Reply:
We can add that as a separate PR later on, so I'll go ahead and merge this.